### PR TITLE
Simplify condition to define ID as autoincrement or sequence in liquibase change log

### LINKED ISSUE #10684: autoIncrement variable in initial_schema.xml is not defined for Postgres
##### **Overview of the issue**

I created a microservice and selected Postgres as database type. It created `00000000000000_initial_schema.xml` file and it refers to a `${autoIncrement}` in the xml twice, but that variable is not defined anywhere. I looked at the `initial_schema.xml.ejs` file in JHipster source code and it seems like it creates the `autoIncrement` property only for MySQL and MariaDB:

```
<%_ if (prodDatabaseType === 'mysql' || prodDatabaseType === 'mariadb') { _%>
    <property name="autoIncrement" value="true"/>
<%_ } _%>
```

But it still refers to `${autoIncrement}` for microservices using Postgres. It doesn't cause an error but it's confusing. Am I missing something? 
##### **Reproduce the error**

Create a microservice and select Postgres as the database type.

##### **JHipster Version(s)**

6.3.1

##### **JHipster configuration**

<details><summary>jhipster info</summary>
<p>
INFO! Using JHipster version installed locally in current project's node_modules
INFO! Executing jhipster:info
INFO! Options: from-cli: true
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
oct-25-eclx-3@0.0.0 /Users/eciftci/workspace/oct25eclx3
└── generator-jhipster@6.3.1

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp"
    },
    "jhipsterVersion": "6.3.1",
    "applicationType": "microservice",
    "baseName": "oct25eclx3",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8081",
    "authenticationType": "oauth2",
    "cacheProvider": "no",
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Memory",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "gradle",
    "enableSwaggerCodegen": true,
    "embeddableLaunchScript": false,
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [],
    "enableTranslation": false,
    "clientPackageManager": "npm",
    "blueprints": [],
    "skipClient": true,
    "skipUserManagement": true
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>

##### **Environment and Tools**

openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)

git version 2.21.0

node: v12.1.0

npm: 6.9.0

yeoman: 3.1.0

yarn: 1.15.2

Docker version 19.03.4, build 9013bf5

docker-compose version 1.24.1, build 4667896b
</p>
</details>

##### **Browsers and Operating System**
MacOS Catalina

-   [x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by ersinciftci**
Fixes #10684 

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/templates/src/main/resources/config/liquibase/changelog/initial_schema.xml.ejs [hunk 1]
```diff
@@ -24,10 +24,10 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
                         http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
 
-    <%_ if (prodDatabaseType === 'mysql' || prodDatabaseType === 'mariadb') { _%>
+    <%_ if (prodDatabaseType === 'mysql' || prodDatabaseType === 'mariadb' || prodDatabaseType === 'postgresql') { _%>
     <property name="autoIncrement" value="true"/>
     <%_ } _%>
-    <%_ if (prodDatabaseType === 'postgresql' || prodDatabaseType === 'oracle' || prodDatabaseType === 'mssql') { _%>
+    <%_ if (prodDatabaseType === 'oracle' || prodDatabaseType === 'mssql') { _%>
```
Comment (pmverma) [inline generators/server/templates/src/main/resources/config/liquibase/changelog/initial_schema.xml.ejs]: Having a false value for other DBs is better, including Postgres.

**Comment by ersinciftci** [comment]
The failing test is also failing on master branch, so I think this PR can be merged.

**Comment by ersinciftci** [comment]
Rebased with master and all tests are passing now

**Comment by DanielFran** [comment]
Thanks @ersinciftci for your 1st contribution

**Pull request merged**